UI/window-basic-main.cpp

```diff
@@ -1782,7 +2158,7 @@ void OBSBasic::CreateHotkeys()
 	LoadHotkeyPair(streamingHotkeys,
 			"OBSBasic.StartStreaming", "OBSBasic.StopStreaming");
 
-	auto cb = [] (void *data, obs_hotkey_id, obs_hotkey_t*, bool pressed)
+	auto cb = [](void *data, obs_hotkey_id, obs_hotkey_t*, bool pressed)
```

Comment:
There's some unrelated whitespace change here that doesn't belong in this commit